src/corefx/System.Security.Cryptography.Native/openssl.c

```diff
@@ -623,3 +655,123 @@ GetX509NameInfo(
 
     return NULL;
 }
+
+/*
+Function:
+GetX509StackFieldCount
+
+Used by System.Security.Cryptography.X509Certificates' OpenSslX509ChainProcessor to identify the
+number of certificates returned in the built chain.
+
+Return values:
+0 if the field count cannot be determined, or the count of certificates in STACK_OF(X509)
+Note that 0 does not always indicate an error, merely that GetX509StackField should not be called.
+*/
+int
+GetX509StackFieldCount(
+    STACK_OF(X509)* stack)
```

Comment:
It's slightly strange to see this in a signature we plan to P/Invoke to, but I guess it's ok because it's a pointer and so we'll always just use IntPtr on the managed side?